code/modules/mob/living/carbon/human/species/station/traits_vr/negative.dm

```diff
@@ -96,6 +96,30 @@
 	cost = -1
 	var_changes = list("flash_mod" = 2.0)
 
+/datum/trait/hollow
+	name = "Hollow Bones/Aluminum Alloy"
+	desc = "Your bones and robot limbs are much easier to break."
+	cost = -4
+
+/datum/trait/hollow/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
+	for(var/obj/item/organ/external/O in H.organs)
+		if(istype(O, /obj/item/organ/external/head))
+			O.min_broken_damage = 25 //Skulls are a bit denser
+
+		else if(istype(O, /obj/item/organ/external/chest) || istype(O, /obj/item/organ/external/groin))
+			O.min_broken_damage = 20
+
+		else if(istype(O, /obj/item/organ/external/arm) || istype(O, /obj/item/organ/external/leg))
+			O.min_broken_damage = 15
```

Comment:
This stuff is why I suggested doing `min_broken_damage *= 0.75` (or something similar) since that would reduce every organ to 75% of the original value, effectively being a 25% debuff regardless of what the original values were, specifically.

Reply:
The problem is that'd result in weird rounding errors that my OCD dislikes.. Plus, since hollow bones are not uniformly weaker in birds, I feel that it should be reflected in code.

Reply:
No offense but code cleanliness comes before your OCD. Don't use hardcoded organ types. If you really hate non-integer results, put a round(,1) around it.

Reply:
It's more that I don't want a uniform change in bones since the skull is a little more dense than other bones, as well as the fact that the foot/hand starts so low that it requires a bruised damage change too. This would be easier to fine tune in the event that we find that something is too strong or more likely, too weak